src/breadcrumbs/breadcrumbs.scss

```diff
@@ -20,28 +20,27 @@
 
 @mixin iui-breadcrumbs-item {
   display: flex;
-  align-items: center;
-
-  > .iui-anchor {
-    display: inline-block;
-  }
+  line-height: $iui-component-height;
+  height: $iui-component-height;
+  max-width: $iui-3xl * 2;
+  margin: 0 $iui-sm;
 
-  > .iui-anchor,
-  > span,
-  .iui-ellipsis {
-    margin: 0 $iui-sm;
+  > .iui-button {
+    margin: 0 (-$iui-s);
   }
 
-  > .iui-button {
-    margin: 0 $iui-xs;
+  &:last-child {
+    &,
+    > .iui-button {
+      margin-right: 0;
+    }
   }
 
-  > .iui-anchor,
-  > .iui-button > .iui-label {
+  > *,
+  .iui-label {
     overflow: hidden;
     white-space: nowrap;
     text-overflow: ellipsis;
-    max-width: $iui-3xl * 2;
```

Comment:
Why removed?

Reply:
I placed it at the root of breadcrumb-item